cengage/react-magma
# Feat/input label

Issue: # [910](https://github.com/cengage/react-magma/issues/910)

## What I did
Implemented inline left label position for Input.

## Screenshots (if appropriate):
![the-stuff](https://user-images.githubusercontent.com/77400920/196991556-0b00f37a-a625-4d86-a84f-55311dae9f01.png)

## Checklist 

## How to test
Verify if working in following components:
- Text input
- Text area
- Date
- Time
- Combobox
- Native select
- Password
